solc/CommandLineInterface.cpp

```diff
@@ -1341,6 +1343,26 @@ bool CommandLineInterface::assemble(
 				successful = false;
 			else
 				stack.optimize();
+
+			// FIXME: Ugly fragment of code to enable generating ewasm from strict assembly
```

Comment:
This should not be needed after #7797 was merged.

Reply:
e.g. use `solc --yul --yul-dialect evm --machine eWasm`

Reply:
OK. Will remove it.